ts/utils/messages.ts

```diff
@@ -0,0 +1,15 @@
+import { MessageWithContentPO } from "../types/MessageWithContentPO";
+
+/**
+ * A function to compare two messages by date (created_at)
+ *
+ * @param {MessageWithContentPO} m1
+ * @param {MessageWithContentPO} m2
+ * @returns {number}
+ */
+export function messagesComparatorByDateDesc(
+  m1: MessageWithContentPO,
+  m2: MessageWithContentPO
+): number {
+  return Date.parse(m2.created_at) - Date.parse(m1.created_at);
```

Comment:
I fear this sorting is not stable, what happens when two messages have the same timestamp?

Reply:
If compareFunction(a, b) returns 0, leave a and b unchanged with respect to each other, but sorted with respect to all different elements.

Reply:
Actually I believe we can just sort messages by ID, as we use ULID as ID, that provides [lexicographic monotonic IDs](https://github.com/ulid/javascript#monotonic-ulids).
So this should probably be enough:

```ts
return m1.id < m2.id ? 1 : -1;
```

cc @gunzip